### PR TITLE
fix: autocomplete disabled

### PR DESCRIPTION
**Pull request by EdisonSu768**
autocomplete `disabled` not working, when adding this attr.

### CODE REVIEW

File: src/autocomplete/autocomplete.directive.ts [hunk 1]
```diff
@@ -141,6 +142,14 @@ export class AutoCompleteDirective
             this.inputValue$$.next(value);
           }
         });
+
+      this.ngControl.statusChanges
+        .pipe(takeUntil(this.unsubscribe$), startWith(this.ngControl.status))
```
Comment (JounQin) [inline src/autocomplete/autocomplete.directive.ts:147]:
```suggestion
        .pipe(startWith(this.ngControl.status), takeUntil(this.unsubscribe$))
```

File: src/autocomplete/autocomplete.directive.ts [hunk 2]
```diff
@@ -141,6 +142,14 @@ export class AutoCompleteDirective
...
+        .subscribe({
+          next: status => {
+            this.disabled = status === 'DISABLED';
+          },
+        });
```
Comment (JounQin) [inline src/autocomplete/autocomplete.directive.ts:152]:
```suggestion
        .subscribe(status => {
          this.disabled = status === 'DISABLED';
        });
```

**Comment by JounQin** [comment]
@EdisonSu768 添加一下 changeset

File: src/autocomplete/autocomplete.directive.ts (hunk 1 above)
Comment (EdisonSu768) [reply src/autocomplete/autocomplete.directive.ts:147]: sure

File: src/autocomplete/autocomplete.directive.ts (hunk 2 above)
Comment (EdisonSu768) [reply src/autocomplete/autocomplete.directive.ts:152]: copy that

**Comment by EdisonSu768** [comment]
> @EdisonSu768 添加一下 changeset

I will

**Review by JounQin — APPROVED** [review]

**Review by JounQin — APPROVED** [review]

**Pull request merged**